ci-operator/step-registry/ipi/conf/aws/outpost/ipi-conf-aws-outpost-commands.sh

```diff
@@ -0,0 +1,321 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+# EXPIRATION_DATE=$(date -d '4 hours' --iso=minutes --utc)
+# TAGS="Key=expirationDate,Value=${EXPIRATION_DATE}"
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-outpost.yaml.patch
+REGION="${REGION:-$LEASED_RESOURCE}"
+CLUSTER_NAME="$(yq-go r "${CONFIG}" 'metadata.name')"
+
+OUTPOST_ID=$(aws --region "${REGION}" outposts list-outposts | jq -r .Outposts[0].OutpostId)
+OUTPOST_AZ=$(aws --region "${REGION}" outposts list-outposts | jq -r .Outposts[0].AvailabilityZone)
+OUTPOST_ARN=$(aws --region "${REGION}" outposts list-outposts | jq -r .Outposts[0].OutpostArn)
+OUTPOST_INSTANCE_TYPE=$(aws --region "${REGION}" outposts get-outpost-instance-types --outpost-id $OUTPOST_ID | jq -r .InstanceTypes[1].InstanceType)
+
+cat <<_EOF > /tmp/01_vpc.yaml
```

Comment:
Regarding this step, it did too many things, we need to split them into separate steps, please see my comment in https://github.com/openshift/release/pull/32486#discussion_r983052035

Reply:
Why split?  it's just creating a vpc in putpost.  nothing else.